README

```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
 
+This BIDS dataset is part of the [Courtois-Neuromod dataset](https://docs.cneuromod.ca/).
```

Comment:
See my comment above Courtois-Neuromod -> Courtois NeuroMod

Reply:
I would also remove the reference to BIDS from this sentence. Some of our subdataset will be BIDS-like, but not strictly BIDS compliant.